scripts/setup-modules.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -euo pipefail
+
+SRCDIR=$1
+BUILDDIR=$2
+MODULE_PLAYGROUND=$3
+# shellcheck disable=SC2034 # used by the Makefile
+export MODULE_DIRECTORY=$4
+
+# TODO: meson allows only out of tree builds
+if test "$SRCDIR" != "$BUILDDIR"; then
+	rm -rf "$MODULE_PLAYGROUND"
+	mkdir -p "$(dirname "$MODULE_PLAYGROUND")"
+	cp -r "$SRCDIR/$MODULE_PLAYGROUND" "$MODULE_PLAYGROUND"
+fi
+
+"${MAKE-make}" -C "$PWD/$MODULE_PLAYGROUND" modules
```

Comment:
when called from the autotools build, this will honor the parallelism set via -j, whereas it won't for meson. Simple test with meson changing this to include `-j$(nproc)` speeds up the build by a lot on my aged TGL laptop:

command: time ninja -C build-meson test
Before: real    0m13.444s
After: real    0m5.201s

Maybe we should set -j$(nproc) in MAKEFLAGS when calling from meson?